README.md

```diff
@@ -1691,6 +1693,8 @@ hz.getMultiMap('my-distributed-multimap').then(function (mmp) {
 });
 ```
 
+Hazelcast MultiMap uses `EntryListener` to listen to events which occur when entries are added to, or removed from the MultiMap. See the [Entry Listener section](#7522-entry-listener) for information on how to create an entry listener object and register it.
```

Comment:
Hazelcast MultiMap uses `EntryListener` to listen to the events that occur when the entries are added to or removed from the MultiMap. ...